### ISSUE TITLE
Number keys don't work for web script's `keydown` event, even if Vimium has been "unmapAll"

### ISSUE THREAD

**Issue by garywill** [philc/vimium#4233]
**Describe the bug**
I set Vimium to "unmapAll". Number keys can't work on web.

**To Reproduce**
1. Install Vimium-FF 1.67.6 on Firefox 110
2. Open Vimium options. Fill "unmapAll" into "Custom key mappings"
3. Open this html file in Firefox:
    ```html
    <html >
      <head>
        <title>key test</title>
        <script>
            document.addEventListener('keydown', function(evt) {
                console.log(evt);
            });
        </script>
      </head>
      <body>
        key test. show in console
      </body>
    </html>

    ```
5. Press F12 to open console
4. Focus the web page. Press some keys. You'll find `keydown` event can't receive 1-9 number key event

**Comment by gdh1995** [comment]
Yes `unmapAll` doesn't affect `0-9`, but there' still a chance to exclude them - you may add `123456789` as `Pass Keys` of an exclusion rule, like this:
![image](https://user-images.githubusercontent.com/5547703/226184586-333f77e5-67da-40f6-ad90-385c2ac3f191.png)

And then click `Save Options` and refresh pages to make it work.
